openssl/aes.go

```diff
@@ -272,39 +222,28 @@ func (x *aesCTR) XORKeyStream(dst, src []byte) {
 	if len(src) == 0 {
 		return
 	}
-	C._goboringcrypto_EVP_AES_ctr128_enc(
-		x.ctx,
-		(*C.uint8_t)(unsafe.Pointer(&src[0])),
-		(*C.uint8_t)(unsafe.Pointer(&dst[0])),
-		C.size_t(len(src)))
+	C.go_openssl_EVP_EncryptUpdate_wrapper(x.ctx, base(dst), base(src), C.int(len(src)))
 	runtime.KeepAlive(x)
 }
 
 func (c *aesCipher) NewCTR(iv []byte) cipher.Stream {
-	x := &aesCTR{key: c.key}
-	copy(x.iv[:], iv)
```

Comment:
I'm not copying `iv` here because `EVP_CipherInit_ex` already copies it before storing.